data/migrations/run_migrations.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+migration_dir=${0%/*}
+
+# first run functions and triggers, bailing if either of these fail, as they
+# are required by later steps.
+psql --set ON_ERROR_STOP=1 -f "${migration_dir}/../functions.sql" $*
+if [ $? -ne 0 ]; then echo "Installing new functions failed.">&2; exit 1; fi
+psql --set ON_ERROR_STOP=1 -f "${migration_dir}/../triggers.sql" $*
+if [ $? -ne 0 ]; then echo "Installing new triggers failed.">&2; exit 1; fi
+
+# then disable triggers
+for table in planet_osm_point planet_osm_line planet_osm_polygon; do
+    psql -c "ALTER TABLE ${table} DISABLE TRIGGER USER" $*
+done
+
+# run updates in parallel. note that we don't bail here, as we want to
+# re-enable the triggers regardless of whether we failed or not.
+for sql in ${migration_dir}/*.sql; do
```

Comment:
I figured that we could only migrate to the "current" release anyway, because we only keep single copies of `functions.sql` and `triggers.sql`. We could keep older versions of these files, but I figure if I need to migrate to `v0.5.3` then I can always checkout that tag.

Therefore, if we can only migrate to the "current" release, we don't need to version the migration files and we should remove them all in the commit following the release tag by which point they're only of historical interest.

What do you think?